tests/test_recv.py

```
@@ -0,0 +1,172 @@
+		self.recv_tool = multicast.recv.McastRECV()
+		# Store original stdout for later restoration
+		self.original_stdout = sys.stdout
+
+	def tearDown(self):
+		"""Tear down test fixtures."""
+		# Restore original stdout
+		sys.stdout = self.original_stdout
+		super(McastRECVTestSuite, self).tearDown()
+
+	@mock.patch('multicast.recv.module_logger')
+	def test_doStep_with_response(self, mock_logger):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_doStep_with_response`

Add return type annotation: `None`

(ANN201)